rest-service/manager_rest/cluster_status_manager.py

```diff
@@ -100,6 +100,36 @@
         name='Prometheus',
         description='Prometheus monitoring service'),
 }
+SERVICE_ASSIGNMENTS = {
+    CloudifyNodeType.DB: [
+        'etcd',
+        'haproxy',
+        'nginx',
+        'node_exporter',
+        'patroni',
+        'postgres_exporter',
+        'postgresql-9.5',
```

Comment:
This service shouldn't be on a DB node, only on an AIO node.

Reply:
(though we do have other code that handles that, so not necessarily a major concern at this point)

Reply:
yes, exactly my attitude here